static/js/info_overlay.js

```diff
@@ -22,99 +22,147 @@ export let toggler;
 
 const markdown_help_rows = [
     {
-        markdown: "*italic*",
+        markdown: "*" + $t({defaultMessage: "italic"}) + "*",
         usage_html: "(or <kbd>Ctrl</kbd>+<kbd>I</kbd>)",
     },
     {
-        markdown: "**bold**",
+        markdown: "**" + $t({defaultMessage: "Bold"}).toLowerCase() + "**",
```

Comment:
What's the story with capitalizing this? If it's the capitalization linter firing, we should just add this to the exclude list in `tools/lib/capitalization.py`.